legal-api/migrations/versions/5238dd8fb805_.py

```diff
@@ -56,17 +59,74 @@ def upgrade():
         sa.Column('foreign_jurisdiction_region', sa.String(length=10), nullable=True),
         sa.Column('foreign_name', sa.String(length=100), nullable=True),
         sa.Column('foreign_identifier', sa.String(length=50), nullable=True),
+        sa.Column('version', sa.Integer(), nullable=False),
+        sa.Column('change_filing_id', sa.Integer(), nullable=False),
```

Comment:
`change_filing_id` is missing in `amalgamations`, `amalgamations_history`, `amalgamating_businesses_history` tables. 👀  It's used in the new versioning approach.

Reply:
> `change_filing_id` is missing in `amalgamations`, `amalgamations_history`, `amalgamating_businesses_history` tables. 👀 It's used in the new versioning approach.

For other 3 tables, it has `filing_id`, which is the same as `change_filing_id`. Wondering for `amalgamating_businessws_history`, should I rename to `change_filing_id` or keep `filing_id`?

Reply:
I talked to Vysakh a bit and have a better understanding now. 

This is what we'll need:
* `amalgamations`/`amalgamation_history` tables will need `filing_id` and `change_filing_id`
* `amalgamating_businesses`/`amalgamating_businesses_history` will only need `change_filing_id`